Source/DataSources/EllipsoidGeometryUpdater.js

```diff
@@ -322,19 +336,21 @@ import Property from './Property.js';
             options.slicePartitions = slicePartitions;
             options.subdivisions = subdivisions;
             options.offsetAttribute = offsetAttribute;
-            options.radii = in3D ? unitSphere : radii;
-            var innerRadii = Property.getValueOrDefault(ellipsoid.innerRadii, time, radii, new Cartesian3());
-            if (in3D) {
-               var mag = Cartesian3.magnitude(radii);
-               var innerRadiiUnit = new Cartesian3(innerRadii.x/mag, innerRadii.y/mag, innerRadii.z/mag);
-               options.innerRadii = innerRadiiUnit;
+            options.radii = Cartesian3.clone(in3D ? unitSphere : radii, options.radii);
+            if (defined(innerRadii)) {
+                if (in3D) {
+                    var mag = Cartesian3.magnitude(radii);
+                    options.innerRadii = Cartesian3.fromElements(innerRadii.x/mag, innerRadii.y/mag, innerRadii.z/mag, options.innerRadii);
+                } else {
+                    options.innerRadii = Cartesian3.clone(innerRadii, options.innerRadii);
+                }
             } else {
-               options.innerRadii = innerRadii;
+                options.innerRadii = undefined;
```

Comment:
This also looks like an extraneous space.

Reply:
This one was actually improperly indented before